docs/README.md

```diff
@@ -1,33 +1,25 @@
-# Website
+# Reanimated Documentation
 
-This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
+👉 [https://docs.swmansion.com/react-native-reanimated/](https://docs.swmansion.com/react-native-reanimated/)
 
-## Installation
+## What's in the docs
 
-```console
-yarn install
-```
+The docs come with a [Fundamentals](https://reanimated-beta-docs.swmansion.com/docs/fundamentals/getting-started) section which step-by-step introduction into the world of Reanimated.
```

Comment:
```suggestion
The docs come with a [Fundamentals](https://reanimated-beta-docs.swmansion.com/docs/fundamentals/getting-started) section which is a step-by-step introduction into the world of Reanimated.
```

Reply:
Would we both get `Co-authored-by` if I had commited your suggestion? 🤔

Reply:
Fixed in ee1389f7ef935ceb59a6bb6487550204c0419b5b (╯°□°)╯︵ ┻━┻